src/test/unit/mocks/mockRequest.ts

```diff
@@ -26,3 +26,29 @@ export const mockRequest = ({
   } as any;
   return req;
 };
+
+export const returnToExistingMockRequest = ({
```

Comment:
You don't need to create a new one, the original object can be extended with new values.